/accept rejomy returns NullPointerException if i dont have any requests

```
[23:59:29 ERROR]: Exception when Welse attempted to tab complete /accept Rejomy
org.bukkit.command.CommandException: Unhandled exception during tab completion for command '/accept Rejomy' in plugin ZonePracticePro v6.21-SNAPSHOT
        at org.bukkit.command.PluginCommand.tabComplete(PluginCommand.java:154) ~[patched_1.8.8.jar:git-PandaSpigot-173]
        at org.bukkit.command.SimpleCommandMap.tabComplete(SimpleCommandMap.java:228) ~[patched_1.8.8.jar:git-PandaSpigot-173]
        at org.bukkit.command.SimpleCommandMap.tabComplete(SimpleCommandMap.java:176) ~[patched_1.8.8.jar:git-PandaSpigot-173]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.tabCompleteCommand(CraftServer.java:1704) [patched_1.8.8.jar:git-PandaSpigot-173]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.tabComplete(CraftServer.java:1672) [patched_1.8.8.jar:git-PandaSpigot-173]
        at net.minecraft.server.v1_8_R3.MinecraftServer.tabCompleteCommand(MinecraftServer.java:1241) [patched_1.8.8.jar:git-PandaSpigot-173]
        at net.minecraft.server.v1_8_R3.PlayerConnection$9.evaluate(PlayerConnection.java:2002) [patched_1.8.8.jar:git-PandaSpigot-173]
        at net.minecraft.server.v1_8_R3.PlayerConnection$9.evaluate(PlayerConnection.java:1999) [patched_1.8.8.jar:git-PandaSpigot-173]
        at org.bukkit.craftbukkit.v1_8_R3.util.Waitable.run(Waitable.java:24) [patched_1.8.8.jar:git-PandaSpigot-173]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:839) [patched_1.8.8.jar:git-PandaSpigot-173]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:412) [patched_1.8.8.jar:git-PandaSpigot-173]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:756) [patched_1.8.8.jar:git-PandaSpigot-173]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:648) [patched_1.8.8.jar:git-PandaSpigot-173]
        at java.base/java.lang.Thread.run(Thread.java:1447) [?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.List.iterator()" because the return value of "java.util.Map.get(Object)" is null
        at dev.nandi0813.practice.Command.Accept.AcceptCommand.onTabComplete(AcceptCommand.java:125) ~[?:?]
        at org.bukkit.command.TabCompleter.onTabComplete(TabCompleter.java:27) ~[patched_1.8.8.jar:git-PandaSpigot-173]
        at org.bukkit.command.PluginCommand.tabComplete(PluginCommand.java:142) ~[patched_1.8.8.jar:git-PandaSpigot-173]
        ... 13 more
```

--- a/core/src/main/java/dev/nandi0813/practice/Command/Accept/AcceptCommand.java
+++ b/core/src/main/java/dev/nandi0813/practice/Command/Accept/AcceptCommand.java
@@ -117,14 +117,22 @@ public class AcceptCommand implements CommandExecutor, TabCompleter {
 
         if (args.length == 1) {
             if (profile.isParty()) {
-                for (PartyRequest partyRequest : PartyManager.getInstance().getRequestManager().getRequests().get(PartyManager.getInstance().getParty(player))) {
-                    if (!arguments.contains(partyRequest.getSender().getLeader().getName()))
-                        arguments.add(partyRequest.getSender().getLeader().getName());
+                List<PartyRequest> requests = PartyManager.getInstance().getRequestManager().getRequests().get(PartyManager.getInstance().getParty(player));
+
+                if (requests != null) {
+                    for (PartyRequest partyRequest : requests) {
+                        if (!arguments.contains(partyRequest.getSender().getLeader().getName()))
+                            arguments.add(partyRequest.getSender().getLeader().getName());
+                    }
                 }
             } else {
-                for (DuelRequest duelRequest : DuelManager.getInstance().getRequests().get(player)) {
-                    if (!arguments.contains(duelRequest.getSender().getName()))
-                        arguments.add(duelRequest.getSender().getName());
+                List<DuelRequest> requests = DuelManager.getInstance().getRequests().get(player);
+
+                if (requests != null) {
+                    for (DuelRequest duelRequest : DuelManager.getInstance().getRequests().get(player)) {
+                        if (!arguments.contains(duelRequest.getSender().getName()))
+                            arguments.add(duelRequest.getSender().getName());
+                    }
                 }
             }
 

